scripts/package/arch-packages/activitywatch-bin/PKGBUILD

```diff
@@ -0,0 +1,57 @@
+# Maintainer: Erik Bjäreholt <erik.bjareholt@gmail.com>
+
+pkgname=activitywatch-bin
+pkgver='0.7.1'
+pkgrel=1
+epoch=
+pkgdesc="Log what you do on your computer. Simple, extensible, no third parties."
+arch=('i686' 'x86_64')
+url="https://github.com/ActivityWatch/activitywatch"
+license=('MPL2')
+groups=()
+depends=()
+makedepends=()
+checkdepends=()
+optdepends=()
+provides=()
+conflicts=("activitywatch")
+replaces=()
+backup=()
+options=()
+install=
+changelog=
+source=("${pkgname}.zip::https://github.com/ActivityWatch/activitywatch/releases/download/v${pkgver}/activitywatch-v${pkgver}-linux-x86_64.zip")
```

Comment:
No need to rename the source to `$pkgname.zip` here, but, more importantly, should keep the version in the filename.

Reply:
Alright, so `$pkgname-$pkgver.zip`?